### ISSUE TITLE
iOS 18 IMAP fails to see any messages/folders

### ISSUE THREAD

**Issue by tarko** [mguessan/davmail#369]
iOS default Mail client fails to see any emails or folders after upgrade to iOS 18. No error is shown.

I have not looked into it in detail but I found https://github.com/stalwartlabs/mail-server/issues/765#issuecomment-2362931057 that seems pretty similar. Could this be the problem with davmail as well?

**Comment by esabol** [comment]
That's very interesting, @tarko. Could you post a WIRE debug log of an interaction between DavMail and iOS 18? Check the first FAQ here:

https://davmail.sourceforge.net/faq.html#Feedback

If you prefer to edit your properties file yourself, put the following line in your properties file:
```
log4j.logger.httpclient.wire=DEBUG
log4j.logger.org.apache.http.wire=DEBUG
```
Quit and restart the DavMail application. Then have your mail program connect to DavMail and try to authenticate again. DavMail will then create a detailed log of what it is doing. This log will be stored in a `davmail.log` file. By default, it will be located in DavMail's current working directory on Linux and Windows or in `~/Library/Logs/DavMail/davmail.log` on macOS. (The location of this file can be customized with the `Log File Path` setting.) This log file might include information like your password and other account details that you should not share online. If you are confident you can remove those things from the davmail.log file, feel free to post the sanitized file here. Otherwise, send the log file to [mguessan@free.fr](mailto:mguessan@free.fr) along with the URL of this GitHub issue. I hope this helps!

**Comment by tarko** [comment]
Thanks, I unicasted the trace directly to you, had to add some more logging to capture the IMAP side as well.

I don't see nothing immediate from the logs that screams "I'm broken"

**Comment by esabol** [comment]
Just to clarify, I am not [mguessan@free.fr](mailto:mguessan@free.fr). That's the project owner and principal programmer. I'm just a user of DavMail who likes to follow the development of DavMail and help other users of DavMail when and where I can. 😄

**Comment by tarko** [comment]
Yeah, my bad. Either way the logs are in correct place and I also continue looking into this. At least it is not some TLS etc issue as IMAP connection is established just fine. I also found some other cries about iOS 18 IMAP so there is hope that the bug is actually on iOS side and will hopefully be fixed in 18.1 due soon.

**Comment by tarko** [comment]
FYI this has not been fixed by IOS 18.1 nor 18.2 :(

**Comment by esabol** [comment]
Is this also a problem with Apple Mail on the latest macOS, too?

**Comment by tarko** [comment]
Sorry no idea, not a mac user

**Comment by mguessan** [comment]
Patch merged, closing

**Comment by esabol** [comment]
@tarko asked:
> Any chance for a release?

The pull request for this fix is in DavMail 6.3.0. Please try that.

**Comment by tarko** [comment]
nevermind, trunk builds are available, problem solved :)

**Comment by esabol** [comment]
@tarko wrote:
> nevermind, trunk builds are available, problem solved :)

And the 6.3.0 release didn't?
